Added Artifact_Examples (#170)
rapid7/insightconnect-workflows: workflows/Artifact_Examples/help.md

# Description

> This uses an API Trigger with each of the DataTypes. The workflow then has different Actions to see how to work with that data.

Reviewer: ```suggestion
This workflow lets you explore how to use various data types within InsightConnect. It uses an API Trigger for manual input for each of the available types and then runs different actions to show how they work and finally prints a resulting artifact card displaying the values.
```

# Key Features

* Examples of how to present data in Artifacts
* No Plugins or Orchestrator needed, good for training
* All in the Cloud so execution is fast

# Requirements

> none

Reviewer 2: ```suggestion
None
```

# Documentation

## Setup

Once the workflow has been downloaded, login to InsightConnect and "Import" it into the workflow builder. There are intentionally no connections to configure and does not need an Orchestrator because no Plugins are used.

## Technical Details

> Plugins utilized by workflow: none

Reviewer 2: ```suggestion
|Plugin|Version|Count|
|----|----|--------|
```

## Troubleshooting

_There is no troubleshooting information at this time_

# Version History

* 1.0.0 - Initial workflow

# Links

## References

> * https://github.com/rapid7/insightconnect-workflows

Reviewer 2: ```suggestion
* [InsightConnect Workflows](https://github.com/rapid7/insightconnect-workflows)
```
